data-loading/src/test/java/no/sikt/nva/data/report/api/etl/SingleObjectDataLoaderTest.java

```diff
@@ -222,18 +230,23 @@ private static void catchExpectedExceptionsExceptHttpException(Exception e) {
         }
     }
 
-    private String generatePropertyTriple(IndexDocumentWithConsumptionAttributes updatedIndexDocument) {
+    private String generatePropertyTriple(IndexDocumentWithConsumptionAttributes document) {
         return String.format("<" + UriWrapper.fromHost(HOST)
-                                       .addChild(updatedIndexDocument.indexDocument().identifier().toString())
+                                       .addChild(NVI_CANDIDATES_FOLDER)
```

Comment:
NVI_CANDIDATES_FOLDER is a bit misleading to read here, even though it doesn't matter. Rename constant to NVI_PATH and give some other value?